scripts/get-cloud-observations.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+"""Gets observations from PANOPTES Observation Portal [https://github.com/panoptes/panoptes-tom].
+
+This Python script calls the 'get-cloud-observations' cloud function to
+generate a list of observation targets from the portal.
+This target list is then outputted to a YAML file.
+
+Example:
+python scripts/get-cloud-observations.py --targets-file resources/targets/simple_test.yaml \
+     --facility PAN001 --overwrite
+
+"""
+
+import click
+import yaml
+
+import subprocess
+
+from panoptes.pocs.utils.logger import get_logger
+from panoptes.utils.serializers import from_json, to_json
+
+logger = get_logger()
+
+
+@click.command()
+@click.option("--cloud-function-name", default="get-tom-observations")
+@click.option("--targets-file", required=True, help="File for writing observation debug.")
+@click.option("--facility", required=True, help="Observation facility name (ie. PAN001)")
+@click.option(
+    "--overwrite", is_flag=True, help="Should targets file be appended to or overwritten."
+)
+def main(
+    targets_file=None, facility=None, cloud_function_name="get-tom-observations", overwrite=False,
+):
+    # Convenience function for printing if verbose.
+    logger.info(f"Calling {cloud_function_name}, writing new targets to {targets_file}")
+
+    # Call the cloud function get json data.
+    new_targets = call_cloud_function(cloud_function_name, facility)
+
+    # Create targets.
+    target_list = list()
+    for target in new_targets:
+        if target["facility"] == facility:
+            logger.info(f"Adding {target}")
+            # Pull out relevant fields.
+            data = target["parameters"]
+
+            ra = data.get("field_ra")
+            dec = data.get("field_dec")
+            position = f"{ra}deg {dec}deg"
+
+            target_info = dict(
+                name=data.get("name"),
+                position=position,
+                priority=data.get("priority", 100),
+                exp_time=data.get("exp_time"),
+                min_nexp=data.get("min_nexp"),
+                exp_set_size=data.get("exp_set_size"),
```

Comment:
This might handle the default values oddly.  Might be better to just not include them in the dictionary unless there is a value.

Reply:
We could check this by trying to actually create `astropy.coordinates.SkyCoord` object (or a POCS `Field` and `Observation`) from each `target_info` before you `append` it.  If it fails then just skip it?

It will handle that during the intake on the scheduler side, but it might be nice for the script user to know they downloaded ones that won't work.